### PR TITLE
Update uk.coffee

### PR DESCRIPTION
**Pull request by AlenaSSobolieva**
Hello)
I have tried to make a translation into Ukrainian. If it is OK, I will continue)

### CODE REVIEW

**Comment by nwinter** [comment]
Thanks @AlenaSSobolieva! Can you [sign the CLA](https://codecombat.com/cla)?

**Comment by AlenaSSobolieva** [comment]
> Thanks @AlenaSSobolieva! Can you [sign the CLA](https://codecombat.com/cla)?

I have just sighed it)))

File: app/locale/uk.coffee [hunk 1]
```diff
@@ -12,38 +12,38 @@ module.exports = nativeDescription: "Українська", englishDescription:
...
 #    speech_bubble_testimonial_1_blockquote: "Once again, your team has managed to accomplish the Holy Grail of Edutainment. In my opinion as a 26-year veteran Middle School educator, there is no more engaging resource for teaching coding."
 #    core_curriculum_header: "Core Computer Science Curriculum"
 #    core_curriculum_p: "We have developed our instructional model based on proven theories and extensive evidence around how students learn best."
 #    core_curriculum_carousel_item1_header: "Make Connections"
 #    core_curriculum_carousel_item1_p: "At the start of every module, use our turnkey lesson slides to help your students make the connection between new Computer Science concepts and relevant examples from their daily lives."
 #    core_curriculum_carousel_item2_header: "Construct Own Meaning"
 #    core_curriculum_carousel_item2_p: "Next, guide your students in constructing a deeper understanding of new concepts through unplugged and discussion activities that are built into our lesson plans."
-#    core_curriculum_carousel_item3_header: "Learn New Concepts"
+#    core_curriculum_carousel_item3_header: "Вивчи нові поняття" # {change}
```
Comment (mrfinch) [inline app/locale/uk.coffee:23]: you actually need to remove `#` and not add `# {change}` in the translations

**Review by mrfinch — COMMENTED** [review]
Apologies for not reviewing but we plan to use some tool to translate so hopefully we can get these out faster for you using that. Thanks

**Pull request closed without merging**